paig-examples/python-openai-ai-app/README.md

```diff
@@ -0,0 +1,32 @@
+# Simple AI Application demonstration using PAIG
+
+
+## Step 1: Start the PAIG server
+If you have not started the PAIG server, please follow the instructions in the [PAIG Quick Start Guide](https://docs.paig.ai/index.html) to start the PAIG server.
```

Comment:
We should have specific instructions for how start the PAIG server in standalone mode